guides/source/layout.html.erb

```diff
@@ -39,59 +43,72 @@
         <li class="more-info"><a href="https://github.com/rails/rails">Contribute on GitHub</a></li>
       </ul>
     </div>
-  </div>
-  <div id="header">
+  </nav>
+  <header id="page_header">
     <div class="wrapper clearfix">
-      <div class="header-logo"><a href="index.html" title="Return to home page">Guides.rubyonrails.org</a></div>
-      <ul class="nav">
-        <li><a class="nav-item" href="index.html">Home</a></li>
-        <li class="guides-index guides-index-large">
-          <a href="index.html" id="guidesMenu" class="guides-index-item nav-item">Guides Index</a>
-          <div id="guides" class="clearfix" style="display: none;">
-            <hr />
-            <dl class="guides-section-container">
-              <% documents_by_section.each do |section| %>
-                <div class="guides-section">
-                  <dt><%= section['name'] %></dt>
+      <nav id="feature_nav">
+        <div class="header-logo">
+          <a href="index.html" title="Return to the Guides Home for <%= @edge.present? ? 'Edge' : @version %> Guides">Guides</a>
+          <span id="version_switcher">
+            Version:
+            <select class="guides-version">
+              <option value="https://edgeguides.rubyonrails.org/"<%= " selected" if @edge %>>Edge</option>
+              <% %w[7.1 7.0 6.1 6.0 5.2 5.1 5.0 4.2 4.1 4.0 3.2 3.1 3.0 2.3].each do |version| %>
+                <option value="https://guides.rubyonrails.org/v<%= version %>/"<%= " selected" if @version&.start_with?(version) %>><%= version %></option>
+              <% end %>
+            </select>
+          </span>
+        </div>
+        <ul class="nav">
+          <li><a class="nav-item" id="home_nav" href="https://rubyonrails.org/">Home</a></li>
+          <li class="guides-index guides-index-large">
```

Comment:
Not sure how much control we have over the columns here, I noticed on Safari the "Policies" group/guides are kept on the middle column, which causes it to go all the way to the bottom of the container, and at a certain size one guide breaks the line and makes the column overflow a bit. It doesn't seem to be a problem on Chrome because that same group is moved to the 3rd column, so while there's a bit of line breaking and/or height adjusting as you resize, it doesn't make it overflow.

<details><summary>Safari</summary>
<p>

https://github.com/rails/rails/assets/26328/9e5f85f5-1b59-4fc6-a91f-7f6ced67c048

</p>
</details> 


<details><summary>Chrome</summary>
<p>

https://github.com/rails/rails/assets/26328/57e5eb2e-89dc-47f6-a50a-db88c088201e

</p>
</details> 

Just noting it here, in case there's something you think we can do to prevent it. Not a blocker.

Reply:
Reducing the guides-section-container height to 53em from 55em kicks policies to the next column for safari. 81277220